aws-event-bridge/src/main/scala/akka/stream/alpakka/aws/eventbridge/scaladsl/EventBridgePublisher.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.aws.eventbridge.scaladsl
+
+import akka.stream.alpakka.aws.eventbridge.EventBridgePublishSettings
+import akka.stream.scaladsl.{Flow, Keep, Sink}
+import akka.{Done, NotUsed}
+import software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient
+import software.amazon.awssdk.services.eventbridge.model._
+
+import scala.concurrent.Future
+import scala.compat.java8.FutureConverters._
+
+/**
+ * Scala API
+ * Amazon Event Bridge publisher factory.
+ */
+object EventBridgePublisher {
+
+  /**
+   * creates a [[akka.stream.scaladsl.Flow Flow]] to publish a message to an EventBridge using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
+   */
+  def flow(settings: EventBridgePublishSettings = EventBridgePublishSettings())(
+      implicit eventBridgeClient: EventBridgeAsyncClient
+  ): Flow[PutEventsRequestEntry, PutEventsResponse, NotUsed] =
+    Flow
+      .fromFunction((message: PutEventsRequestEntry) => PutEventsRequest.builder().entries(message).build())
+      .via(publishFlow(settings))
+
+  /**
+   * creates a [[akka.stream.scaladsl.Flow Flow]] to publish a message to an EventBridge using an [[software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient EventBridgeAsyncClient]]
+   */
+  def flowSeq(settings: EventBridgePublishSettings = EventBridgePublishSettings())(
+      implicit eventBridgeClient: EventBridgeAsyncClient
+  ): Flow[Seq[PutEventsRequestEntry], PutEventsResponse, NotUsed] =
+    Flow
+      .fromFunction((messages: Seq[PutEventsRequestEntry]) => PutEventsRequest.builder().entries(messages: _*).build())
```

Comment:
It looks like sending a batch of events isn't all or nothing.  Some may succeed and some may fail throughout the sequence.  I'm not suggesting we add any logic for handling retries, etc., but we should probably add some notes about this either here or in Alpakka documentation.

https://docs.aws.amazon.com/eventbridge/latest/userguide/add-events-putevents.html

Reply:
Addressed in the documentetaion.